src/SolutionNormPostProcessing.C

```diff
@@ -339,6 +339,12 @@ SolutionNormPostProcessing::execute()
     int offSet = 0;
     for ( size_t j = 0; j < fieldPairVec_.size(); ++j ) {
 
+      // sync fields
+      fieldPairVec_[j].first->modify_on_device();
+      fieldPairVec_[j].first->sync_to_host();
+      fieldPairVec_[j].second->modify_on_device();
+      fieldPairVec_[j].second->sync_to_host();
```

Comment:
Exact DOF field is `user_functions` and don't have NGP implementations. I suppose we are computing exact fields again somewhere after this?

Reply:
@sayerhs Thanks for catching this. You are right, the exact fields should not be modified anywhere on device. I have removed these calls in https://github.com/Exawind/nalu-wind/commit/4911b48139317aaee25402b59baa3fbb4a35a4e6